integrations/extensions/docs/elasticsearch-install-and-setup/python-document-ingestion/README.md

```diff
@@ -0,0 +1,151 @@
+
+```bash
+curl -X PUT "${ES_URL}/_ingest/pipeline/${ES_PIPELINE_NAME}?pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+-H "Content-Type: application/json" --cacert "${ES_CACERT}" -d'
+{
+  "processors": [
+    {
+      "inference": {
+        "model_id": ".elser_model_1",
+        "target_field": "ml",
+        "field_map": {
+          "text": "text_field"
+        },
+        "inference_config": {
+          "text_expansion": {
+            "results_field": "tokens"
+          }
+        }
+      }
+    }
+  ]
+}'
+```	 
+
+
+### Step 4: Create an index for storing tokens as rank features
+
+```bash
+curl -X PUT "${ES_URL}/${ES_INDEX_NAME}?pretty" -u "${ES_USER}:${ES_PASSWORD}" \
+-H "Content-Type: application/json" --cacert "${ES_CACERT}" -d'
+{
+  "settings":{
+  "index":{
+  "default_pipeline":${ES_PIPELINE_NAME}
+  }
+},
+  "mappings": {
+    "properties": {
+      "ml.tokens": {
+        "type": "rank_features"
+      },
+      "text": {
+        "type": "text"
+      }
+    }
+  }
+}'
+```
+
+NOTE: `rank_features` only works for ELSER v1 model. ELSER v2 requires a different type for the tokens. ELSER v2 has only been available since Elastic 8.11. Learn more about ELSER v2 from [here](https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html)
+
+
+### Step 5: Running the python notebook
+
+### Step 5a. Install the required libraries
+
+To get started, navigate to the `python-document-ingestion` directory and install the requirements provided in the `requirements.txt` directory:
+
+```bash
+cd python-document-ingestion
+pip3 install -r requirements.txt
+```
+
+### Step 5b. Run the provided notebook to ingest documents
+
+Start the notebook environment using the below command from within your created `venv` from [Step 1](#step-1-create-a-python-virtual-environment-and-upgrade-pip). Then, open the `doc-ingestion.ipynb` notebook on your web browser.
+
+```bash
+jupyter notebook
+```
+
+Run each cell in order using the "Shift + Enter" shortcut or by clicking the ![play](../assets/jupyter_play_button.png) button.
+
+After the last cell has completed execution, you will see a message like "**Indexed n/n documents**" that indicates that your documents were successfully chunked and ingested into Elasticsearch. If you see an error and only a few of your documents got indexed, you may want to delete the index through the [Kibana dashboard](../ICD_Elasticsearch_install_and_setup.md#step-2-set-up-kibana-to-connect-to-elasticsearch) and restart again by attempting with smaller number of documents at a time.
+
+Your documents are now available in the index, ready for searching and querying. Follow the steps outlined below to use this index in a RAG based setup with Watson Assistant. 
+
+**NOTE**: There are some example documents available [here](https://ibm.ent.box.com/s/l2rhvajs6jasz2rqla7eakz7uml8oljs/folder/230228063107), if you would like to test the setup.
```

Comment:
Will update this link from example docs uploaded in https://github.com/watson-developer-cloud/assistant-toolkit/pull/244/files after it gets merged

Reply:
That one is merged now.

Reply:
Link updated yesterday